docs/_partials/icons/tanstack-start.mdx

```diff
@@ -0,0 +1,6 @@
+<img
+  width="32"
+  height="32"
+  alt="tanstack start logo"
```

Comment:
This is redundant and noisy because the logos are displayed with names/titles already. Can we replace with `alt=""`?